coops-ui/src/views/Dashboard.vue

```diff
@@ -196,18 +197,18 @@ export default {
         this.inProcessFiling = filingId
       }
 
-      // check for updated status to reload dashboard
+      // reset iteration counter
       this.checkFilingStatusCount = 0
+
+      // check for updated status to reload dashboard
       this.checkFilingStatus(filingId)
     },
 
+    // checks whether this filing's status has changed
+    // respawns itself approx. every 1 second for up to 10 iterations
     checkFilingStatus (filingId) {
-      // check whether this filing's status has changed - recursive, runs approx. every 1 second for up to 10 seconds
-
-      this.checkFilingStatusCount++
-
       // stop this cycle after 10 iterations
-      if (this.checkFilingStatusCount > 10) {
+      if (++this.checkFilingStatusCount >= 10) {
```

Comment:
This is interesting. How does this work?

Reply:
In a nutshell, the property is incremented BEFORE it's used in the expression. Compare with post-increment where the property is incremented AFTER it's used in the expression. In this case I felt that it cleaned up the code nicely and made sense.

https://www.google.com/search?q=javascript+pre+increment